frontend/puppeteer/launchPlayers.js

```diff
@@ -0,0 +1,69 @@
+const puppeteer = require('puppeteer-core');
+
+(async () => {
+    const url = "http://localhost:3000/"
+    const browser = await puppeteer.launch({ 
+        executablePath: 'C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe', 
```

Comment:
If it's possible, it would be good to not have a hardcoded file path for the browser as it may be different for different users.